extension/data/frame/frame.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin-top: 1px;
+  font-family: "Lucida Grande", "Segoe UI", Arial, sans-serif;
+  font-size: 11px;
+  color: inherit;
+  text-align: right;
```

Comment:
I would keep it right for now. We can see what people think and may change it later.